x-pack/plugin/esql/qa/testFixtures/src/main/resources/airports.csv

```diff
@@ -0,0 +1,892 @@
+abbrev:keyword,name:text,scalerank:integer,type:keyword,location:geo_point
+LUH,Sahnewal,9,small,POINT(75.9570722403652 30.8503598561702)
```

Comment:
I'd really appreciate if these were aligned for easier scanning. That's not something we tend to do with code, but these data are scanned so much I think it's worth it.

Reply:
I noticed that most other CSV files in these tests did not align columns, but the main employees one did (partially). Anyway I aligned the columns in the new PR.